hockeysdk/src/main/res/values-zh/strings.xml

```diff
@@ -0,0 +1,94 @@
+<resources>
+    <!-- Crash Dialog -->
+    <string name="hockeyapp_crash_dialog_app_name_fallback">此应用</string>
+    <string name="hockeyapp_crash_dialog_title">%s 意外终止</string>
+    <string name="hockeyapp_crash_dialog_message">您愿意发送一份匿名报告以便我们修复问题？</string>
+    <string name="hockeyapp_crash_dialog_negative_button">不发送</string>
+    <string name="hockeyapp_crash_dialog_neutral_button">总是发送</string>
+    <string name="hockeyapp_crash_dialog_positive_button">发送报告</string>
+
+    <!-- Download Failed -->
+    <string name="hockeyapp_download_failed_dialog_title">下载失败</string>
+    <string name="hockeyapp_download_failed_dialog_message">无法下载更新，是否重试？</string>
+    <string name="hockeyapp_download_failed_dialog_negative_button">取消</string>
+    <string name="hockeyapp_download_failed_dialog_positive_button">重试</string>
+
+    <!-- Update -->
+    <string name="hockeyapp_update_mandatory_toast">%s 有更新可用并且为强制更新! </string>
+    <string name="hockeyapp_update_dialog_title">有更新</string>
+    <string name="hockeyapp_update_button">更用</string>
+    <string name="hockeyapp_update_dialog_message">显示关于更新的信息？</string>
+    <string name="hockeyapp_update_dialog_negative_button">忽略</string>
+    <string name="hockeyapp_update_dialog_positive_button">显示</string>
+
+    <!-- Expiry Info -->
+    <string name="hockeyapp_expiry_info_title">版本过期</string>
+    <string name="hockeyapp_expiry_info_text">%s 已经过期，请在 HockeyApp 中检查是否存在更新版本。</string>
+
+    <!-- Feedback Activity -->
+    <string name="hockeyapp_feedback_failed_title">挂起</string>
```

Comment:
This translation looks odd. I was trying to understand the context and then come up with a better one, but checked with @ranterle and the string does not seem to be used, so we may just leave it as it is or remove it.